components/embl-logo/assets/logo-medium.svg

```diff
@@ -1 +1 @@
2000/svg" viewBox="0 0 170.64 74.48"><defs><style>.cls-2{fill:#009f4c}</style></defs><path d="M17.4 46.44H2.96v-8.07h13.07v-3.05H2.96v-7.28h14.08V25.1H0v24.28h17.4v-2.94zm27.95 2.94V25.1H41s-1 2.46-1.08 2.75l-5.78 17.51h-.07l-5.85-17.51c-.09-.3-1.08-2.75-1.08-2.75h-4.35v24.28h3V35c0-.71-.07-3.58-.07-5.84h.07l6.67 20.2h3.27l6.66-20.23h.07c0 2.29-.07 20.23-.07 20.23z"/><path d="M54.13 27.82h6.12c3.69 0 5.2 1.3 5.2 3.58 0 3-2.06 3.85-5.23 3.85h-6.09zm-3 21.56h10.35c5.43 0 8-3.2 8-7.12 0-4.2-2.49-5.33-4.09-6 2.7-1.26 2.93-3.51 2.93-5.11 0-3.35-2-6.07-6.58-6.07H51.16zm3-11.32h6.63c2.7 0 5.85.68 5.85 4.13 0 2.73-1.85 4.47-5.3 4.47h-7.18z" fill-rule="evenodd"/><path d="M77.48 25.1h-3.2v24.28h15.97v-3.05H77.48V25.1z"/><path class="cls-2" d="M137.11 0a3.78 3.78 0 103.78 3.78A3.78 3.78 0 00137.11 0zm29.71 16.77a3.78 3.78 0 103.79 3.78 3.79 3.79 0 00-3.79-3.78zm-9.77-5.62a3.78 3.78 0 103.78 3.79 3.79 3.79 0 00-3.78-3.79zm9.81 39.1a3.78 3.78 0 103.78 3.75 3.79 3.79 0 00-3.78-3.75zm-29.75-39.1a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.15a3.79 3.79 0 103.78 3.79 3.79 3.79 0 00-3.78-3.79zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.15a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.15a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm19.83-33.46a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm-.01 11.16a3.78 3.78 0 103.79 3.78 3.78 3.78 0 00-3.79-3.78zm.01 11.15a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm-.01 11.15a3.78 3.78 0 103.79 3.78 3.79 3.79 0 00-3.79-3.78zm-19.82 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zM127.22 5.6A3.79 3.79 0 10131 9.39a3.79 3.79 0 00-3.78-3.79zm0 11.17a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.16A3.78 3.78 0 10131 54a3.79 3.79 0 00-3.78-3.75zm.01 11.19a3.79 3.79 0 103.77 3.79 3.79 3.79 0 00-3.77-3.79zM147 5.6a3.79 3.79 0 103.78 3.79A3.79 3.79 0 00147 5.6zm0 11.17a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm19.85-11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zM147 50.25a3.78 3.78 0 103.81 3.75 3.79 3.79 0 00-3.81-3.75zm0 11.19a3.79 3.79 0 103.78 3.79 3.79 3.79 0 00-3.78-3.79zm-39.6-44.67a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.75 3.78 3.78 0 00-3.78-3.75zm9.91-39.1a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.15a3.79 3.79 0 103.78 3.79 3.79 3.79 0 00-3.78-3.79zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78z"/><path d="M117.31 44.61a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78z" fill="#e40046"/><path class="cls-2" d="M117.31 55.76a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78z"/></svg>
```

Comment:
the state of this SVG file 😭 - I don't think we'd need the classnames or the ids? I tried running this through [SVGOMG](https://jakearchibald.github.io/svgomg/) but it didn't show as an SVG :(

Reply:
let's get this in, and I can look at fixing the Adobe-ified SVG stuff later

Reply:
I dare not tread within ... at least until I know no further revisions to the logo. 😭